recipe/meta.yaml

```diff
@@ -36,12 +36,13 @@ requirements:
   host:
     - pthread-stubs
     - xorg-kbproto
-    - xorg-libx11 1.6.*
+    - xorg-libx11 >=1.6,<2
+    - xorg-libx11
```

Comment:
Maybe I'm missing something, but is it correct to have two different lines listing `xorg-libx11` here?

Reply:
Hi @pkgw! Yes, I've told Felix to include it this way.
If we'd (for whatever reason) include `xorg-libx11` in the global pinnings at some point, then `conda-build` will pick up a version-constraint-less entry to apply the pinning to. For entries that have constraints, (like the `>=1.6` here) `conda-build` will not apply (potential) pinnings too.

TL;DR: `- xorg-libx11 >=1.6,<2` is for us to declare a minimum version we want to use for the build and `- xorg-libx11` is for `conda-build` to apply pinnings to.

Reply:
Ah, I wasn't aware of that pattern. The intention is certainly non-obivous — would one of you mind adding a comment to the `meta.yaml` briefly explaining what's going on?

Reply:
Happy to!
I'd rather have more than less of such explanatory comments anyway :).
Let me know if you'd want me to clarify things further!